### PR TITLE
New: Add the ability to add new classes for any document type [ED-7811]

### PR DESCRIPTION
**Pull request by TzviRabinovitch**

### CODE REVIEW

File: core/base/document.php [hunk 1]
```diff
@@ -320,7 +320,7 @@ public function get_container_attributes() {
 			}
 		}
 
-		return $attributes;
+		return apply_filters( 'elementor/document/container_attributes', $attributes, $this );
```
Comment (udidol) [inline core/base/document.php:323]: Please add doc for this explaining it

**Review by udidol — CHANGES_REQUESTED** [review]
Check PHPLint. Not sure why it is suddenly failing.

**Comment by StuartElementor** [comment]
I merged in a fix for that php lint issue .all checks passed.

File: core/base/document.php (hunk 1 above)
Comment (TzviRabinovitch) [reply core/base/document.php:324]: Done

File: core/base/document.php [hunk 2]
```diff
@@ -320,7 +320,8 @@ public function get_container_attributes() {
 			}
 		}
 
-		return $attributes;
+		// apply this filter to allow the attributes to be modified by different sources
```
Comment (udidol) [inline core/base/document.php:323]:
```suggestion
		/**
		 * Document Container Attributes
		 *
		 * This filter allows modifying the HTML attributes on the document's wrapping tag
		 *
		 * @param array $attributes
		 * @param Document $this
		 */
```

**Review by udidol — APPROVED** [review]

**Review by udidol — APPROVED** [review]

**Pull request merged**